net-nntp/suck/suck-4.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools
+
+DESCRIPTION="Grab news from a remote NNTP server and feed them to another"
+HOMEPAGE="https://lazarus-pkgs.github.io/lazarus-pkgs/suck.html"
+SRC_URI="https://github.com/lazarus-pkgs/suck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ssl perl"
+
+DEPEND="sys-libs/db
+	perl? ( dev-lang/perl )
+	ssl? ( dev-libs/openssl )"
```

Comment:
At least openssl is linked to, so this needs to go into `RDEPEND` (and `DEPEND` can inherit `RDEPEND` not to duplicate the line). I also saw `sys-libs/gdbm.5` be linked against, so same there. Both dependencies need to trigger rebuild of suck on subslot upgrade, in case of openssl append `:0=` (another slot exists there but we need 0), for gdbm append `:=` to achieve that.